src/components/application_manager/test/CMakeLists.txt

```diff
@@ -202,6 +206,18 @@ endif()
   file(COPY sdl_preloaded_pt.json DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
   file(COPY sdl_pt_update.json DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 
+  if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+    link_directories (C:/SQL)
```

Comment:
@OHerasym 
Is it possible to replace hardcoded path with e.g. some environment variable?